crocket/src/main/java/com/crocket/Application.java

```diff
@@ -1,22 +1,35 @@
 package com.crocket;
-import java.awt.Color;
-import javax.swing.*; 
+import javax.swing.*;
+
+
 
-public class Application {
-    public static void main( String[] args )
-    {
-        init(); 
-    }
-    private static void init(){
-        JFrame frame = new JFrame("Crocket");
 
-        ImageIcon img = new ImageIcon("crocket/assets/textures/JFrame_Icon.jpg");
-        frame.setIconImage(img.getImage());
 
-        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.setLocationRelativeTo(null);
-        frame.setExtendedState(JFrame.MAXIMIZED_BOTH); 
-        frame.setVisible(true);  
+public class Application extends JComponent{
+
+    static CroquetView frame = new CroquetView();
+      
+    static Ball ball = new Ball(19,19,100,200,20);
+    static CroquetController cc = new CroquetController(frame, ball);
```

Comment:
Might should the controller have the frame? We might just want to give it an ball view.